cmd/builder/internal/config/default.yaml

```diff
@@ -0,0 +1,28 @@
+dist:
+  module: go.opentelemetry.io/collector/cmd/otelcorecol
+  name: otelcorecol
+  description: Local OpenTelemetry Collector binary, testing only.
+  version: 0.56.0-dev
+  otelcol_version: 0.56.0
```

Comment:
More of a question for Core approvers/maintainers, but I think updating this file would need to become part of the release process right?

Reply:
Would be cool if ocb was able to use the [releases manifest](https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol/manifest.yaml).

Reply:
> updating this file would need to become part of the release process right

Yep, it gets updated as part of the `prepare-release` make target.

Reply:
It looks like that target knows to update `/cmd/builder/internal/builder/config.go`, but would need updated to know to update these files.

Reply:
Sorry, I'm not following you here. `make config` in the cmd/builder directory updates cmd/builder/internal/builder/config/default.yaml, which is then embedded in cmd/builder/internal/builder/config/default.go.

Reply:
I missed that you also updated the  root Makefile 👍

Reply:
Does this need to be updated now that 0.57.2 is out?